src/solvers/dgsem_tree/indicators_2d.jl

```diff
@@ -125,6 +125,292 @@ function (indicator_hg::IndicatorHennemannGassner)(u::AbstractArray{<:Any,4},
 end
 
 
+function (indicator_hg::IndicatorHennemannGassner)(u::AbstractArray{<:Any,4}, mesh::Union{UnstructuredMesh2D},
```

Comment:
What's the difference to the `TreeMesh`?

Reply:
e.g. no AMR. Changed now in 3bad0d4.